src/generator/LevelGenerator.cpp

```diff
@@ -181,6 +186,29 @@ LevelGenerator::Generate() {
           }
           objs->push_back(std::make_shared<gameobject::Platform>(
               glm::vec3(xPos, yPos, zPos), glm::vec3(power, .5f, 7.0f)));
+          if ((rand() % 2) + 1 == 1) {
```

Comment:
This is more of a design decision, but I think as long as the difficulty doesn't vary too greatly (and we don't make impossible levels), making the level have variable elements makes it more exciting. just my opinion though. team meeting? 👍